src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -35,7 +35,7 @@ public class AddressBookParser {
      */
     private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
     private static final Pattern GET_COMMAND_FORMAT = Pattern
-            .compile("(?<commandWord>[get\\s]+)(?<prefix>[/a-z]+)(?<arguments>[a-zA-Z0-9\\s]*)");
+            .compile("(?<commandWord>[get\\s]+)(?<prefix>[/a-z]+)(?<arguments>[a-zA-Z-0-9\\s]*)");
```

Comment:
Regex now accepts literal "-" character to prevent user inputs of negative numbers from registering as "Unknown Command"